docs/1wire.md

```diff
@@ -26,17 +26,17 @@ Currently supported on the SPRACINGF3, STM32F3DISCOVERY, NAZE32 (including clone
 
 ## Usage
 
-  - Plug in the USB cable and connect to your board with the CleanFlight configurator.
+  - Plug in the USB cable and connect to your board with the INAV configurator.
 
     - For boards without a built in USB/UART adapter, you'll need to plug an external one in. Here is how you wire up the CC3D. Plug your USB/UART adapter into the Flexi port:
 
       ![Flashing BlHeli Bootloader](assets/images/serial1wire-cc3d-wiring.jpg)
 
   - Open the BlHeli Suite.
 
-  - Ensure you have selected the correct Atmel or SILABS "Cleanflight" option under the "Select ATMEL / SILABS Interface" menu option.
+  - Ensure you have selected the correct Atmel or SILABS "INAV" option under the "Select ATMEL / SILABS Interface" menu option.
```

Comment:
This is not correct. In BLHeli it's still called Cleanflight, there's no INAV option.